### PR TITLE
feat(WebexMember): display exernal user's organization

### PR DESCRIPTION
**Pull request by karinasigartau0798**
Updated WebexMember to display the external user's organization name in WebexMemberRoster if the member is not from the same organization as the current user.

https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-220418

### CODE REVIEW

File: src/components/WebexMember/WebexMember.jsx [hunk 1]
```diff
@@ -22,16 +22,18 @@ export default function WebexMember({
...
-
   const isMuted = member && member.muted;
-
   return (
     <div className={`${WEBEX_COMPONENTS_CLASS_PREFIX}-member`}>
       <WebexAvatar personID={personID} displayStatus={displayStatus} />
-      <div className="member-name">{`${firstName} ${lastName}`}</div>
+      <div className="member-name">
+        <div>{`${firstName} ${lastName}`}</div>
+        {ID !== personID && destinationType === DestinationType.ROOM && member && member.orgID !== orgID && <div className="member-organization">{`${member.orgID}`}</div>}
```
Comment (maloun96) [inline src/components/WebexMember/WebexMember.jsx:35]: is this feature available only for `DestinationType.ROOM`? Or it can be also for `DestinationType.MEETING`?

File: src/components/WebexMemberRoster/WebexMemberRoster.stories.js [hunk 2]
```diff
@@ -10,7 +10,7 @@ const Template = (args) => <WebexMemberRoster {...args} />;
 
 export const Space = Template.bind({});
 Space.args = {
-  destinationID: 'room1',
+  destinationID: 'room2',
   destinationType: 'room',
 };
```
Comment (maloun96) [inline src/components/WebexMemberRoster/WebexMemberRoster.stories.js:15]: I would suggest adding a new story for this.


`export const DifferentOrganization = Template.bind({});
DifferentOrganization.args = {
  destinationID: 'room2',
  destinationType: 'room',
};`

File: src/components/WebexMember/WebexMember.jsx [hunk 3]
```diff
@@ -21,17 +21,19 @@ export default function WebexMember({
   personID,
   displayStatus,
 }) {
-  const {firstName, lastName} = usePerson(personID);
+  const {firstName, lastName,orgID} = usePerson(personID);
```
Comment (maloun96) [inline src/components/WebexMember/WebexMember.jsx:24]:
```suggestion
  const {firstName, lastName, orgID} = usePerson(personID);
```

File: src/components/WebexMember/WebexMember.jsx [hunk 4]
```diff
@@ -21,17 +21,19 @@ export default function WebexMember({
...
-
   const isMuted = member && member.muted;
-
   return (
     <div className={`${WEBEX_COMPONENTS_CLASS_PREFIX}-member`}>
       <WebexAvatar personID={personID} displayStatus={displayStatus} />
-      <div className="member-name">{`${firstName} ${lastName}`}</div>
+      <div className="member-name">
+        <div>{`${firstName} ${lastName}`}</div>
+        {me.orgID !== orgID && <div className="member-organization">{`${orgID}`}</div>}
```
Comment (maloun96) [inline src/components/WebexMember/WebexMember.jsx:35]:
```suggestion
        {me.orgID !== orgID && <div className="member-organization">{orgID}</div>}
```

File: src/components/WebexMember/WebexMember.jsx (hunk 1 above)
Comment (cipak) [reply src/components/WebexMember/WebexMember.jsx]: good question. @lalli-flores, we decided to implement this for meeting, too, as this is how the desktop client behaves.

Comment (lalli-flores) [reply src/components/WebexMember/WebexMember.jsx]: Yeah, works for me

File: src/components/WebexMemberRoster/WebexMemberRoster.stories.js (hunk 2 above)
Comment (lalli-flores) [reply src/components/WebexMemberRoster/WebexMemberRoster.stories.js:15]: I think for this scenario, there should be an existing story from the Roster PR but it would get updated here because of the changes to member

File: src/components/WebexMember/WebexMember.jsx [hunk 5]
```diff
@@ -31,7 +32,10 @@ export default function WebexMember({
   return (
     <div className={`${WEBEX_COMPONENTS_CLASS_PREFIX}-member`}>
       <WebexAvatar personID={personID} displayStatus={displayStatus} />
-      <div className="member-name">{`${firstName} ${lastName}`}</div>
+      <div className="description">
+        <div className="name">{`${firstName} ${lastName}`}</div>
+        {me.orgID !== orgID && <div className="organization">{orgID}</div>}
+        </div>
```
Comment (cipak) [inline src/components/WebexMember/WebexMember.jsx:38]: indentation

File: src/components/WebexMember/WebexMember.jsx [hunk 6]
```diff
@@ -31,7 +32,10 @@ export default function WebexMember({
   return (
     <div className={`${WEBEX_COMPONENTS_CLASS_PREFIX}-member`}>
       <WebexAvatar personID={personID} displayStatus={displayStatus} />
-      <div className="member-name">{`${firstName} ${lastName}`}</div>
+      <div className="description">
```
Comment (cipak) [inline src/components/WebexMember/WebexMember.jsx:35]: please use `details` to align with the other PRs

File: src/components/WebexMember/WebexMember.jsx [hunk 7]
```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-
-import {usePerson, useMembers} from '../hooks';
+import {DestinationType} from '@webex/component-adapter-interfaces'
```
Comment (cipak) [inline src/components/WebexMember/WebexMember.jsx:3]: is `DestinationType` still needed?

File: src/styles/_variables.scss [hunk 8]
```diff
@@ -24,6 +24,8 @@ $md-red-60: #D4371C !default;
 
 $md-black-100: #000000;
 
+$md-yellow-40: #FC9D03 !default;
```
Comment (cipak) [inline src/styles/_variables.scss:27]: yellow-40 is already in this file (line 21), together with the grays. Maybe it was added by a recent merge.

File: src/components/WebexMember/WebexMember.jsx [hunk 9]
```diff
@@ -37,6 +40,7 @@ export default function WebexMember({
       <WebexAvatar personID={personID} displayStatus={displayStatus} />
       <div className="details">
         <div className="name">{`${firstName} ${lastName}`}</div>
+        {isExternal && <div className="organization">{organization.name}</div>}
```
Comment (lalli-flores) [inline src/components/WebexMember/WebexMember.jsx:43]: This should be displayed under roles 

```
_________
|        | <Person name>
| Avatar | <Role> (e.g. You, presenter, etc.)
|        | <Org if external>
----------

File: src/components/WebexMember/WebexMember.scss [hunk 10]
```diff
@@ -24,7 +29,7 @@
   .presenter {
     @extend .subtext;
   }
-
+
```
Comment (lalli-flores) [inline src/components/WebexMember/WebexMember.scss:32]: Careful of unnecessary whitespace. Please remove

File: src/components/WebexMemberRoster/WebexMemberRoster.stories.js (hunk 2 above)
Comment (lalli-flores) [reply src/components/WebexMemberRoster/WebexMemberRoster.stories.js:16]: @karinasigartau0798, by this I meant that you should not modify data and/or stories in the roster component. Please revert this changes

File: src/data/memberships.js [hunk 11]
```diff
@@ -88,6 +88,13 @@ export default {
         sharing: false,
         inMeeting: true,
       },
+      {
```
Comment (lalli-flores) [inline src/data/memberships.js:91]: Is there a reason for modifying this membership?

**Review by lalli-flores — CHANGES_REQUESTED** [review]
Some comments to address but good otherwise!
Somethings went wrong with your commit rebasing as it says the commit authored by the publishing bot. Not a big deal but if you can fix it, it would be great

File: src/components/WebexMemberRoster/WebexMemberRoster.stories.js (hunk 2 above)
Comment (karinasigartau0798) [reply src/components/WebexMemberRoster/WebexMemberRoster.stories.js]: Done.

File: src/components/WebexMember/WebexMember.jsx (hunk 9 above)
Comment (karinasigartau0798) [reply src/components/WebexMember/WebexMember.jsx]: Done.

File: src/components/WebexMember/WebexMember.scss (hunk 10 above)
Comment (karinasigartau0798) [reply src/components/WebexMember/WebexMember.scss]: Done.

File: src/data/people.js [hunk 12]
```diff
@@ -25,7 +25,7 @@ export default {
     lastName: 'Edwards',
     nickname: 'Gio',
     avatar: './images/giacomo.png',
-    orgID: 'org1',
+    orgID: 'org2',
```
Comment (lalli-flores) [inline src/data/people.js:28]: You're already using `user5` [in your Storybook story](https://github.com/webex/components/pull/185/files#diff-c43f541e598ddf02dd93bc360731acb3b3b7406d8e8d331ace9b880d7250a21eR34), no need to make another user "external". Please revert changes on `user2`

**Review by lalli-flores — COMMENTED** [review]
Some changes to the user data and we should be good to go

File: src/data/people.js (hunk 12 above)
Comment (karinasigartau0798) [reply src/data/people.js]: Done.

**Review by lalli-flores — APPROVED** [review]
